### PR TITLE
DOC-425 part 7: VarScan2, VarScan2 Annotation, VCF LiftOver

### PR DESCRIPTION
**Pull request by ebusygina**
Drafts for VarScan2, VarScan2 Annotation, VCF LiftOver workflows

### CODE REVIEW

File: docs/Encyclopedia/pages/VCF_LiftOver.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# VCF LiftOver
+
+## Description ##
+
+VCF LiftOver is a pipeline used in GDC whole genome sequencing (WGS) harmonization.
+
+## Overview ##
+
+VCF LiftOver is a pipeline used for WGS harmonization of genomic profiling reports. Harmonization is performed with VCF LiftOver using tumor and normal alignments and generates combined nucleotide variation data.
```
Comment (wwysoc2) [inline docs/Encyclopedia/pages/VCF_LiftOver.md:9]:
```suggestion
VCF LiftOver is a pipeline used for harmonization of genomic profiling reports and generates combined nucleotide variation data.
```

File: docs/Encyclopedia/pages/VCF_LiftOver.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+### Input
+
+* Submitted sequencing reads
+
+### Output
+
+* VCF (Data Type: Raw CGI Variant)
+
+## References ##
```
Comment (wwysoc2) [inline docs/Encyclopedia/pages/VCF_LiftOver.md:19]:
```suggestion
```

File: docs/Encyclopedia/pages/VCF_LiftOver.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+1. []()
```
Comment (wwysoc2) [inline docs/Encyclopedia/pages/VCF_LiftOver.md:21]:
```suggestion
```

Comment (wwysoc2) [reply docs/Encyclopedia/pages/VCF_LiftOver.md:21]: not sure if removing it completely works with suggestions.

File: docs/Encyclopedia/pages/VarScan2.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
+# VarScan2
+
+## Description ##
+
+VarScan2 is a somatic variant calling pipeline used in GDC whole exome sequencing (WXS) and targeted sequencing harmonization.
+
+## Overview ##
+
+VarScan2 is one of the four pipelines used for WXS and targeted sequencing somatic variant calling at the GDC. Somatic variant calling is performed with VarScan2 using tumor and normal alignments and generates single-nucleotide polymorphism (SNP) and insertion/deletion (indel) data.
```
Comment (wwysoc2) [inline docs/Encyclopedia/pages/VarScan2.md:9]:
```suggestion
VarScan2 is one of the four pipelines used for WXS and targeted sequencing somatic variant calling at the GDC. Somatic variant calling is performed with VarScan2 using tumor and normal alignments and generates single-nucleotide polymorphism (SNP) data.
```

**Review by wwysoc2 — CHANGES_REQUESTED** [review]

File: docs/Encyclopedia/pages/VarScan2.md (hunk 4 above)
Comment (ebusygina) [reply docs/Encyclopedia/pages/VarScan2.md]: changed

File: docs/Encyclopedia/pages/VCF_LiftOver.md (hunk 2 above)
Comment (ebusygina) [reply docs/Encyclopedia/pages/VCF_LiftOver.md]: deleted the References section

File: docs/Encyclopedia/pages/VCF_LiftOver.md (hunk 3 above)
Comment (ebusygina) [reply docs/Encyclopedia/pages/VCF_LiftOver.md]: deleted the References section

File: docs/Encyclopedia/pages/VCF_LiftOver.md (hunk 1 above)
Comment (ebusygina) [reply docs/Encyclopedia/pages/VCF_LiftOver.md]: changed, thank you for the comment!

File: docs/Encyclopedia/pages/VCF_LiftOver.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+VCF LiftOver is a pipeline used in GDC whole genome sequencing (WGS) harmonization.
+
+## Overview ##
+
+VCF LiftOver is a pipeline used for harmonization of genomic profiling reports and generates combined nucleotide variation data.
+
+### Input
+
+* Submitted sequencing reads
```
Comment (wwysoc2) [inline docs/Encyclopedia/pages/VCF_LiftOver.md:13]:
```suggestion
* Submitted genomic profiles
```

**Review by wwysoc2 — APPROVED** [review]
I forgot the same exact change as the other PR.  Otherwise, LGTM, go ahead and merge.

File: docs/Encyclopedia/pages/VCF_LiftOver.md (hunk 5 above)
Comment (ebusygina) [reply docs/Encyclopedia/pages/VCF_LiftOver.md]: changed

**Pull request merged**